server/modules/playlist_driver.js

```diff
@@ -20,7 +20,14 @@ const random = require('random-js')();
 const time = require('server/util/time');
 const wallGeometry = require('server/util/wall_geometry');
 
-const driveStateMachine = (playlist, layoutSM) => {
+let timer;
+
+const driveStateMachine = (playlist, layoutSM, clearTimer) => {
+  clearTimer = clearTimer || false;
+  if (clearTimer) {
```

Comment:
This is sort hacky... you might consider putting a comment here to fix this for realz at some future time.